recipes/marisa-trie/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{%set name = "marisa-trie" %}
```

Comment:
I think that providing that library is an open question. Could we build as both `marisa-trie` AND `python-marisa-trie`? Then, if the C++ library gets built, we can supercede `marisa-trie`.